app/views/admin/merchants/index.html.erb

```diff
@@ -1,3 +1,46 @@
+<header>
+  <h1 class="main-title">Little Esty Shop</h1>
+
+  <div class="sub-header">
+    <h2 class="page-title">Admin Dashboard</h2>
+      <nav class="navigation">
+      <%= link_to "Dashboard", admin_index_path %>
+      <%= link_to "Merchants", admin_merchants_path %>
+      <%= link_to "Invoices", admin_invoices_path %>
+    </nav>
+  </div>
+</header>
+
+<section id="enabled_merchants">
+  <ul>
+    <h4>Enabled Merchants</h4>
+    <% @enabled_merchants.each do |merchant| %>
+      <li><%= link_to "#{merchant.name}", admin_merchant_path(merchant) %>
+        <%= form_with model: merchant, url: admin_merchant_path(merchant), local: true do |form| %>
+          <%= form.hidden_field :method, value: "patch" %>
+          <%= form.hidden_field :status, value: "disabled" %>
+          <%= form.submit "Disable #{merchant.name}" %>
+        <% end %>
+      </li>
+    <% end %>
+  </ul>
+</section>
+
+<section id="disabled_merchants">
+  <ul>
+    <h4>Disabled Merchants</h4>
+    <% @disabled_merchants.each do |merchant| %>
+      <li><%= link_to "#{merchant.name}", admin_merchant_path(merchant) %>
+        <%= form_with model: merchant, url: admin_merchant_path(merchant), local: true do |form| %>
+          <%= form.hidden_field :method, value: "patch" %>
```

Comment:
You can put the `patch` method in the `form_with` tag

```
<%= form_with model: merchant, url: admin_merchant_path(merchant), method: "patch", local: true do |form| %>
```

Reply:
actually, I think you can simplify this even further using the `button_to` rails helper.

The way you have it still works, though.

https://apidock.com/rails/ActionView/Helpers/UrlHelper/button_to